manifest-v2.json

```diff
@@ -942,6 +942,14 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
+      "directory": "python3.9/hello-pt",
```

Comment:
This looks incomplete. PR has multiple Python version but the manifest is only updated for py3.9

Reply:
Hi @jfuss! Thanks for the review. Yep, was missing but I added it now.